Ada_Zangemann-cs.txt

```diff
@@ -5,181 +5,123 @@
-Creative Commons licenses are still rare exceptions for commercially 
-published books, and often considered revolutionary and risky by 
-publishers.
-
-<b>A big thank you</b> to <b>dpunkt.verlag</b> and </small><small><b>No 
-Starch Press</b></small><small> for understanding the values of software 
-freedom, taking the time to discuss and understand the advantages of 
-free cultural licenses, and the great support to publish this book under 
-conditions that respect the FSFE's values.
+Tato kniha od Matthiase Kirschnera (autor) a Sandry Brandstätterové (ilustrátorka) je licencována pod licencí <b>Creative Commons CC-BY-SA</b>, což je svobodná kulturní licence.
+
+<b>Můžete</b> sdílet, kopírovat a redistribuovat materiál v jakémkoli médiu nebo formátu. <b>Můžete</b> materiál adaptovat, remixovat, transformovat a vytvářet z něj odvozená díla pro jakýkoli účel, dokonce i komerčně. <b>Musíte</b> uvést správné přisouzení autorství, poskytnout odkaz na licenci a uvést, zda byly provedeny změny.
+<b>Můžete</b> to udělat jakýmkoli přiměřeným způsobem, ale ne způsobem, který by naznačoval, že poskytovatel licence vás nebo vaše použití schvaluje. Pokud remixujete, transformujete nebo vytváříte z materiálu odvozená díla, <b>musíte</b> své příspěvky distribuovat pod stejnou licencí jako původní dílo. (Toto je shrnutí srozumitelné pro lidi – není náhradou za „právní kód“ CC-BY-SA 4.0.)
+
+Navíc u remixovaných verzí (odvozených děl) poskytovatel licence žádá, abyste nijak neměnili osobní poděkování autora a krátké biografie autora a ilustrátorky. Namísto toho prosím tyto části ze své remixované verze úplně odstraňte.
+
+Vydání pod touto svobodnou kulturní licencí bylo možné díky velké podpoře dpunkt.verlag GmbH a O'Reilly Media, Inc. pro německou verzi knihy a No Starch Press pro anglickou verzi. Bohužel, licence Creative Commons jsou stále vzácnou výjimkou u komerčně vydávaných knih a často jsou považovány vydavateli za revoluční a riskantní.
+
+<b>Velké poděkování</b> patří <b>dpunkt.verlag</b> a <small><b>No Starch Press</b></small> za pochopení hodnot svobody softwaru, za čas věnovaný diskusím a pochopení výhod svobodných kulturních licencí a za velkou podporu při vydání této knihy za podmínek, které respektují hodnoty FSFE.
 
 ## 2BB ##
 
-<b>Free as in freedom not price</b>
- 
-Free Software is about the freedoms you receive with the software, not 
-about the price. Likewise publishing a book under a free culture license 
-does not mean you have the right to get a gratis copy of the book or 
-ebook. It means you get the rights described above, which enable society 
-to benefit further from this book, as it should from all cultural works.
-
-We encourage you to support authors, publishers, and others involved in 
-the creation of culture under free licenses – including <b>financially 
-supporting them through purchases or donations</b>. 
+<b>Svobodné jako ve smyslu svobody, ne ceny</b>
```

Comment:
```suggestion
<b>Bez omezení, ne bezplatné</b>
```
Tady mě trochu mrzí, že se tu ztrácí ten wordplay.
Co si myslíte o tomhle?

Reply:
Jo, to není špatné. Možná bych použil příslovce "bez omezení, ne bezplatně"